core/app/models/spree/image/active_storage_attachment.rb

```diff
@@ -3,10 +3,15 @@
 module Spree::Image::ActiveStorageAttachment
   extend ActiveSupport::Concern
   include Spree::ActiveStorageAdapter
+  ACCEPTED_IMAGE_TYPES = %w(image/jpeg image/jpg image/png image/gif).freeze
```

Comment:
What about adding an application configuration for this so that it's easier to configure for stores that want to restrict the list further?

Reply:
On a side note: we can see also how Paperclip is restricting file types and use the same config for Paperclip as well.

Reply:
I replaced both %() in paperclipattachment and activestorage attachment with an not application configuration allowed_image_mime_types.